### ISSUE TITLE
Questions about the PACS dataset experiment results

### ISSUE THREAD

**Issue by C0notSilly** [ExplainableML/ACVC#2]
Hello Ilke, thanks for you great work of single source domain generalization "Attention Consistency on Visual Corruptions for Single-Source Domain Generalization". I'm currently doing some new experiments on the same area, but I have some questions about the experiment data.

1. How do you obtain the domain generalization accuracy results of PACS dataset of the baseline model ? Did you use the best generlization result during training epochs or the average accuracy? 
2. When testing on the PACS dataset, do you test the whole subset of for example art painting or do you test on the test split of the art painting subset ?
I'm looking forward to your timely reply.

**Comment by cuguilke** [comment]
Hi @C0notSilly,

1. We used the final accuracy (at the end of the training) of multiple runs to get avg accuracy and its standard deviation. Number of epochs, like the other hyperparameters, comes from earlier studies done for multi-source domain generalization. Probably not ideal, but we used them to do a fair comparison.
2. For testing on target domains (the things we did not train our model with), we use the full dataset.

I personally don't like PACS, the results often has relatively high standard deviations, making it harder to interpret the results. But, it is widely used in the field. I would prefer using the COCO benchmark we describe in the paper: more classes, ground-truth segmentation masks, and low standard deviation in results.

**Comment by C0notSilly** [comment]
Thank you for thorough reply and valuable advice. Issuse closed.

**Comment by scabcd1234** [comment]
Hello, thank you for your contribution to the Single Source Domain Generalization Institute. I have a question want to ask you, you can compare the precision of the method of M - ADA is how to get, I'm not found in the code for the implementation of this method.
